### PR TITLE
Pagination for student page

### PR DESCRIPTION
**Pull request by komalahire**
Modified in pagination in student page and student class details page.

### CODE REVIEW

File: src/components/Dashboard /PartnerData/index.js [hunk 1]
```diff
@@ -11,26 +12,37 @@ import { toast } from "react-toastify";
 toast.configure();
 
 function PartnerDashboard() {
+  const [pageNumber, setPageNumber] = useState(0);
+
+  const [totalCount, setTotalCount] = useState();
   const [partners, setPartners] = useState([]);
   const [searchTerm, setSearchTerm] = useState("");
-  const [debouncedText] = useDebounce(searchTerm);
+  const [debouncedText] = useDebounce(searchTerm, 1000);
```
Comment (tarun-dugar) [inline src/components/Dashboard /PartnerData/index.js]: reduce to 400 from 1000

File: src/components/Dashboard /StudentClassData/index.js [hunk 2]
```diff
@@ -1,8 +1,21 @@
...
 
 function StudentClassData(props) {
+  const [pageNumber, setPageNumber] = useState(0);
+  const [searchTerm, setSearchTerm] = useState("");
+  const [debouncedText] = useDebounce(searchTerm);
+
+  const usersPerPage = 10;
+  const pagesVisited = pageNumber * usersPerPage;
+
+  const pageCount = Math.ceil(props.location.state.pass.length / usersPerPage);
```
Comment (tarun-dugar) [inline src/components/Dashboard /StudentClassData/index.js:14]: is pass the total number of students?

Comment (komalahire) [reply src/components/Dashboard /StudentClassData/index.js:14]: No we don't have

File: src/components/Dashboard /PartnerData/index.js (hunk 1 above)
Comment (komalahire) [reply src/components/Dashboard /PartnerData/index.js]: Done

**Review by tarun-dugar — APPROVED** [review]

**Pull request merged**